eva-pipeline/src/main/java/embl/ebi/variation/eva/pipeline/annotation/load/VariantAnnotationLineMapper.java

```diff
@@ -0,0 +1,376 @@
+ * 20_60419_A/G	20:60419	G	-	-	-	intergenic_variant	-	-	-	-	-	-
+ * 20_60479_C/T	20:60479	T	-	-	-	intergenic_variant	-	-	-	-	-	rs149529999	GMAF=T:0.0018;AFR_MAF=T:0.01;AMR_MAF=T:0.0028
+ * 20_60523_-/C	20:60522-60523	C	-	-	-	intergenic_variant	-	-	-	-	-	rs150241001	GMAF=C:0.0115;AFR_MAF=C:0.05;AMR_MAF=C:0.0028
+ *
+ * Please note that most of the code is from org.opencb.biodata.formats.annotation.io.VepFormatReader
+ * public methods in VepFormatReader can't be reused because there is a reference to the previous line (currentVariantString)
+ * that prevent each line to be independent
+ *
+ * Here each line is mapped to {@link VariantAnnotation}; in case of two annotations for the same variant, a new
+ * {@link VariantAnnotation} object is created containing only the fields that will be appended:
+ *  - ConsequenceTypes
+ *  - Hgvs
+ */
+public class VariantAnnotationLineMapper implements LineMapper<VariantAnnotation> {
+    private static final Logger logger = LoggerFactory.getLogger(VariantAnnotationLineMapper.class);
+
+    /**
+     * Map a line in VEP output file to {@link VariantAnnotation}
+     * @param line in VEP output
+     * @param lineNumber
+     * @return a {@link VariantAnnotation}
+     * @throws Exception
+     *
+     * Most of the code is from org.opencb.biodata.formats.annotation.io.VepFormatReader#read() with few differences:
+     *  - An empty array is initialized for Hgvs (like ConsequenceTypes);
+     *  - parseFrequencies is always true and the all line is always parsed;
+     *  - The logic to move around the file (read line) and reference to previous line (currentVariantString) are removed;
+     */
+    @Override
+    public VariantAnnotation mapLine(String line, int lineNumber) throws Exception {
+        //logger.debug("Mapping line {} to VariantAnnotation", line);
+        ConsequenceType consequenceType = new ConsequenceType();
+        String[] lineFields = line.split("\t");
+
+        Map<String,String> variantMap = parseVariant(lineFields[0], lineFields[1]);  // coordinates and alternative are only parsed once
+        VariantAnnotation currentAnnotation = new VariantAnnotation(variantMap.get("chromosome"),
+                Integer.valueOf(variantMap.get("start")),
+                Integer.valueOf(variantMap.get("end")), variantMap.get("reference"),
+                variantMap.get("alternative"));
+
+        // Initialize list of consequence types
+        if(currentAnnotation.getConsequenceTypes()==null) {
+            currentAnnotation.setConsequenceTypes(new ArrayList<>());
+        }
+
+        // Initialize Hgvs
+        if(currentAnnotation.getHgvs()==null) {
+            currentAnnotation.setHgvs(new ArrayList<>());
+        }
+
+        /**
+         * parses extra column and populates fields as required. Some lines do not have extra field and end with a \t: the split function above does not return that field
+         * true parameter indicates the function to also parse frequencies
```

Comment:
I think the comment "true parameter indicates the function to also parse frequencies" does not apply any more since there is no reference to the parameter in the code